src/scripts/dungeons/Dungeon.ts

```diff
@@ -9101,7 +9105,10 @@ dungeonList['Moor of Icirrus'] = new Dungeon('Moor of Icirrus',
     [
         new DungeonBossPokemon('Seismitoad', 48000000, 100),
         new DungeonBossPokemon('Whiscash', 48000000, 100),
-        new DungeonBossPokemon('Keldeo', 50000000, 100),
+        new DungeonBossPokemon('Keldeo', 50000000, 100, {
+            hide: false,
+            requirement: new MultiRequirement([new ObtainedPokemonRequirement('Cobalion'), new ObtainedPokemonRequirement('Terrakion'), new ObtainedPokemonRequirement('Virizion')]),
```

Comment:
Since they'll be caught as part of the quest, this can be changed to just require the complete quest